### PR TITLE
Retry once for 302 response.

### PR DESCRIPTION
**Pull request by dkocher**

### CODE REVIEW

File: ctera/src/main/java/ch/cyberduck/core/ctera/CteraSession.java [hunk 1]
```diff
@@ -112,7 +112,7 @@ protected DAVClient connect(final Proxy proxy, final HostKeyCallback key, final
         final HttpClientBuilder configuration = builder.build(proxy, this, prompt);
         configuration.disableRedirectHandling();
         configuration.setServiceUnavailableRetryStrategy(new CustomServiceUnavailableRetryStrategy(host,
-                new ExecutionCountServiceUnavailableRetryStrategy(authentication)));
+                new ExecutionCountServiceUnavailableRetryStrategy(1, authentication)));
```
Comment (ylangisc) [inline ctera/src/main/java/ch/cyberduck/core/ctera/CteraSession.java:115]: Default in `ExecutionCountServiceUnavailableRetryStrategy` is already `1`.

File: ctera/src/main/java/ch/cyberduck/core/ctera/CteraSession.java [hunk 2]
```diff
@@ -111,8 +110,7 @@ public CteraSession(final Host host, final X509TrustManager trust, final X509Key
     protected DAVClient connect(final Proxy proxy, final HostKeyCallback key, final LoginCallback prompt, final CancelCallback cancel) {
         final HttpClientBuilder configuration = builder.build(proxy, this, prompt);
         configuration.disableRedirectHandling();
-        configuration.setServiceUnavailableRetryStrategy(new CustomServiceUnavailableRetryStrategy(host,
-                new ExecutionCountServiceUnavailableRetryStrategy(authentication)));
+        configuration.setServiceUnavailableRetryStrategy(new CustomServiceUnavailableRetryStrategy(host, 1, authentication));
```
Comment (ylangisc) [inline ctera/src/main/java/ch/cyberduck/core/ctera/CteraSession.java:113]: `Math.max(new HostPreferences(host).getInteger("connection.retry"), 1)` ?

Comment (dkocher) [reply ctera/src/main/java/ch/cyberduck/core/ctera/CteraSession.java:113]: Do not expect another 302 response from the server.

**Review by ylangisc — COMMENTED** [review]
No change for CTERA session needed anymore?

**Comment by dkocher** [comment]
> No change for CTERA session needed anymore?

Added test to show new execution count customization option.

**Review by ylangisc — APPROVED** [review]

**Pull request merged**